.circleci/workflows.yml

```diff
@@ -562,6 +562,27 @@ jobs:
           root: .
           paths:
             - ./*
+  fe_version_build:
```

Comment:
can we replace `fe` by `front` or `ui` ?
It takes me few minutes to translate `fe` in `front end` :)

Reply:
Done 🚀